src/main/resources/db/migration/all/20221205160719__add_index_offence_fields_to_ap_applications.sql

```diff
@@ -0,0 +1,5 @@
+TRUNCATE TABLE applications CASCADE;
```

Comment:
This makes me a bit nervous. Can we just set them to a non-sensible default? I just worry about unintended consequences.

Reply:
Sure thing, have updated